common/helpers/sign.c

```diff
@@ -133,8 +131,6 @@ IOTA_EXPORT flex_trit_t* iota_flex_sign_address_gen(
   iss_kerl_key_digest(key, key, key_length, &kerl);
   iss_kerl_address(key, key, security * HASH_LENGTH_TRIT, &kerl);
 
-  kerl_reset(&kerl);
```

Comment:
These resets are needed to make sure that we don't leak key data.

Reply:
It's about cleaning up the memory after storing secrets in it (zeroing it out)

Reply:
right, sorry. btw should we zero out signature too?

Reply:
no, the signature is public anyway, so we don't need to worry about that one.